mteb/abstasks/TaskMetadata.py

```diff
@@ -165,6 +165,20 @@
 METRIC_NAME = str
 METRIC_VALUE = Union[int, float, dict[str, Any]]
 
+
+class DatasetDict(TypedDict, total=False):
+    """A dictionary containing the dataset path and revision.
+
+    Args:
+        path: The path to the dataset.
+        revision: The revision of the dataset.
+        trust_remote_code: Whether to trust the remote code.
+    """
+    path: str
+    revision: str
+    trust_remote_code: bool
```

Comment:
does this then require a trust_remote_code?

Reply:
No, `total=False` means that not all parameters can be passed to the dict